src/control/lib/control/system.go

```diff
@@ -235,6 +236,7 @@ func SystemQuery(ctx context.Context, rpcClient UnaryInvoker, req *SystemQueryRe
 	pbReq.Hosts = req.Hosts.String()
 	pbReq.Ranks = req.Ranks.String()
 	pbReq.Sys = req.getSystem(rpcClient)
+	set mask ear..pbReq.NotJoinedOnly = req.NotJoinedOnly
```

Comment:
Weird typo here?

Reply:
I managed to get some commits mixed up between PRs, resolved and force pushed, thanks.